docs/developer/address.mdx

```diff
@@ -177,3 +177,18 @@ Each field is validated separately, and each issue has a corresponding error cod
 
 - `postalCode` with the wrong format throws the `INVALID` error
 - missing fields mentioned in `requiredFields` throw the `REQUIRED` error
+
+## Skipping the address validation
+
+Since version 3.19, Saleor may completely skip the address validation. To do so, the `AddressInput` must receive `skipValidation` flag set to `true`.
```

Comment:
Since version 3.19, Saleor can skip address validation entirely. To enable this, set the `skipValidation` flag to `true` in the `AddressInput`. This disables all validation except for the `country` field, which still requires a valid `CountryCode` enum.